easy-install.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python3
+import argparse
+import subprocess
+import os
+import sys
+import time
+from shutil import which
+import platform
+from secrets import token_bytes
+from base64 import b64encode
+
+CRED = "\033[31m"
+CEND = "\033[0m"
+CGRN = "\33[92m"
+CYLW = "\33[93m"
+
+
+def clone_frappe_docker_repo() -> None:
+    try:
+        subprocess.run(
+            ["git", "clone", "https://github.com/frappe/frappe_docker"], check=True
+        )
+    except Exception as e:
+        print(f"\n{CRED}Cloning frappe_docker Failed{CEND}\n\n", e)
+
+
+def write_to_env(wd, site,db_pass,email):
+    site_name = site if site != "" else ""
+    with open(os.path.join(wd, ".env"), "w") as f:
+        f.writelines(
+            [
+                "FRAPPE_VERSION=v14.17.1\n",
+                "ERPNEXT_VERSION=v14.9.0\n",
```

Comment:
ummm hardcoding versions?

Reply:
Instead of hardcoding tags we can hard-code branches which don't change frequently. 

There's `v14` tag on docker hub which points to latest images in v14.x.x

https://hub.docker.com/layers/frappe/erpnext-worker/v14/images/sha256-500eb8a09edb45a698b42c1b42eb42441a7e8c1e44ac08d4c0746a72cd4526e5?context=explore

Reply:
These are already part of frappe_docker/example.env, and we have a release action bumping `example.env` on every frappe or erpnext release. Just copying `example.env` to `.env` can get the latest version without hardcoding.